### DISCUSSION TITLE
How can I build raylib-lua with raymob?

### DISCUSSION THREAD

**Discussion by asumbek** [TSnake41/raylib-lua#26] [Q&A]
Hello, I wanted to package my game into Android. So I found [Raymob](https://github.com/Bigfoot71/RayMob) which mostly the thing I need. But I don't know how to configure it with raylib-lua. Do I need to just replace the Raylib submodule with Raymob?

**Comment by lambtoken** [comment]
You can’t just replace the Raylib submodule with Raymob. Raymob is a Raylib fork for Android, so you need to:

- Clone and build Raymob for Android following its instructions.
- Update raylib-lua’s build to link against your Raymob build.
- Adjust Lua bindings if there are differences.
